### PR TITLE
Documentation misc 18 x docs

### PR DESCRIPTION
**Pull request by hvacengi**
A few documentation changes to go with various changes in version 18.

Most notably, I added greater detail to the documentation for cooked steering to help users with tuning.  This information is now reflected in the cooked steering documentation rather than just the steering manager class.

### CODE REVIEW

File: doc/source/commands/flight/cooked.rst [hunk 1]
```diff
@@ -61,7 +67,62 @@ If you ``LOCK`` the ``THROTTLE`` or ``STEERING``, be aware that this prevents th
...
+
+    // MAXSTOPPINGTIME tells kOS how to calculate the maximum allowable
+    // angular velocity.  Increasing the value will result in the ship turning
+    // faster, but it may introduce more overshoot.
+    // Adjust this setting if you have a small amount of torque on a large mass,
+    // or if your ship appears to oscillate back and forth rapidly without
+    // moving towards the target direction.
+    SET STEERINGMANAGER:MAXSTOPPINGTIME TO 10.
+
+    // You can also modify the PID constants for that calculate desired angular
```
Comment (Dunbaratu) [inline doc/source/commands/flight/cooked.rst]: grammar:  "for that calculate"...

File: doc/source/commands/flight/cooked.rst [hunk 2]
```diff
@@ -61,7 +67,62 @@ If you ``LOCK`` the ``THROTTLE`` or ``STEERING``, be aware that this prevents th
 
 When the program ends, these automatically unlock as well, which means that to control a craft you must make sure the program doesn't end. The moment it ends it lets go of the controls.
 
+Tuning cooked steering
+----------------------
+
+While cooked steering tries to come balanced to perform decently without user
+interaction, there are some instances where you may need to help tune the
+behavior.  There are two parts of the steering calculation that can be tuned.
```
Comment (Dunbaratu) [inline doc/source/commands/flight/cooked.rst]: @hvacengi : 
None of the explanation that follows will mean much to a user unless they happen to know that the steering system is one PID feeding into another PID, and which settings are on the "first PID" and which are on the "second PID".  That information is not really communicated clearly in the docs, and was a large part of why I had such trouble with the system.  It wasn't until I read the C# code that I could figure out this really important fact.  Some problems simply cannot be fixed at all if you are adjusting the wrong one of the two levels of PID.

I know you plan to put this stuff into a diagram eventually, but in the meantime just a paragraph or two here would help (or better yet, in the steeringmanager structure page and just link to that explanation paragraph from here)

The paragraph should say that for each of the 3 rotational axes, the system uses one PID controller that seeks an orientation by choosing a rotational velocity, and that this feeds into a second PID controller that seeks that rotational velocity by choosing the controls setting.

Keeping them separate and explaining which suffix is dealing with which of the two steps is really important to helping people tune the system.  Without that they're basically going to be just guessing what to change and not understanding why it's not working in some cases to change the thing they want to change.

I know that you do say several times that "this suffix affects the rotational PID...." but that doesn't really get across the nested PID nature of the system, and which of those two nested PID's is feeding into the other.

If you like, I could have a go at trying to write such a thing up, for your approval, or I could leave it to you.  I have the experience of being someone who was trying to understand your system secondhand so I sort of know where the points of confusions were, but that doesn't mean I know if everything I'm saying is true.

Comment (hvacengi) [inline doc/source/commands/flight/cooked.rst]: Honestly, I think I'd like to have you take a shot at it. I'm normally
pretty good at regrading things, but I think it's apparent that I'm too
close to the implementation to recognize which parts don't come naturally.
But if it'll be to much trouble for you I'll take another shot. I don't
want to take all your time fixing my docs.

On Fri, Nov 13, 2015, 20:14 Steven Mading <notifications@github.com> wrote:

> In doc/source/commands/flight/cooked.rst
> <https://github.com/KSP-KOS/KOS/pull/1247#discussion_r44851315>:
>
> > @@ -61,7 +67,62 @@ If you ``LOCK`` the ``THROTTLE`` or ``STEERING``, be aware that this prevents th
> >
> >  When the program ends, these automatically unlock as well, which means that to control a craft you must make sure the program doesn't end. The moment it ends it lets go of the controls.
> >
> > +Tuning cooked steering
> > +----------------------
> > +
> > +While cooked steering tries to come balanced to perform decently without user
> > +interaction, there are some instances where you may need to help tune the
> > +behavior.  There are two parts of the steering calculation that can be tuned.
>
> @hvacengi <https://github.com/hvacengi> :
> None of the explanation that follows will mean much to a user unless they
> happen to know that the steering system is one PID feeding into another
> PID, and which settings are on the "first PID" and which are on the "second
> PID". That information is not really communicated clearly in the docs, and
> was a large part of why I had such trouble with the system. It wasn't until
> I read the C# code that I could figure out this really important fact. Some
> problems simply cannot be fixed at all if you are adjusting the wrong one
> of the two levels of PID.
>
> I know you plan to put this stuff into a diagram eventually, but in the
> meantime just a paragraph or two here would help (or better yet, in the
> steeringmanager structure page and just link to that explanation paragraph
> from here)
>
> The paragraph should say that for each of the 3 rotational axes, the
> system uses one PID controller that seeks an orientation by choosing a
> rotational velocity, and that this feeds into a second PID controller that
> seeks that rotational velocity by choosing the controls setting.
>
> Keeping them separate and explaining which suffix is dealing with which of
> the two steps is really important to helping people tune the system.
> Without that they're basically going to be just guessing what to change and
> not understanding why it's not working in some cases to change the thing
> they want to change.
>
> I know that you do say several times that "this suffix affects the
> rotational PID...." but that doesn't really get across the nested PID
> nature of the system, and which of those two nested PID's is feeding into
> the other.
>
> If you like, I could have a go at trying to write such a thing up, for
> your approval, or I could leave it to you. I have the experience of being
> someone who was trying to understand your system secondhand so I sort of
> know where the points of confusions were, but that doesn't mean I know if
> everything I'm saying is true.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/KSP-KOS/KOS/pull/1247/files#r44851315>.
>
-- 
-Brad

Comment (Dunbaratu) [inline doc/source/commands/flight/cooked.rst]: I'll give it a try tonight.  I'll pull your PR to my local and then make a few more edits to push back at you for your approval.

**Pull request merged**